### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by MikeTsunami** [APeters97/CS-435-Project-2#3]
https://github.com/APeters97/CS-435-Project-2/blob/c2f400725b67b576426f1cb43bbaed47ca036b63/Test.java#L176-L187

Directed Graph Test seems to take long as well (worst-case time complexity == O(n * (m + k))), where n is the method's input, m is the number of nodes in newDAG.vertices, and k is randNum

**Comment by APeters97** [comment]
Yes, this was addressed in a previous fix in 142de3af95e2c4ceabaa42fe55cb48c58a31e0b5
